.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: ruby
+rvm:
+  - 2.0
+  - 2.1
```

Comment:
If you look closely it says 2 instead of 2.0 for the  Ruby version.  This
is because of the String thing in the  yml.
On Feb 16, 2015 6:40 PM, "Federico Simoncelli" <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/abonas/kubeclient/pull/13#discussion_r24783134>:
>
> > @@ -0,0 +1,8 @@
> > +language: ruby
> > +rvm:
> > +  - 2.0
> > +  - 2.1
>
> It seems fine to me:
> https://travis-ci.org/simon3z/kubeclient/builds/50911341
>
> ruby 2.0.0p598 (2014-11-13 revision 48408) [x86_64-linux]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/abonas/kubeclient/pull/13/files#r24783134>.
>